apps/alpha-test-app/src/components/molecules/FormDemoMolecule.vue

```diff
@@ -0,0 +1,102 @@
+<script lang="ts" setup>
+import { TestInput } from "sit-onyx";
+import { ref, watch } from "vue";
+
+type Props = {
```

Comment:
```suggestion
type FormData = {
```

The type is only for a single prop, not for all :D